generator/src/main/java/com/scottlogic/deg/generator/GenerateTestCases.java

```diff
@@ -99,14 +99,13 @@ public void run() {
         GenerationConfig config = new GenerationConfig(this);
 
         try {
-            DataGeneratorMonitor monitor = new NoopDataGeneratorMonitor();
+            DataGeneratorMonitor monitor = new SystemOutDataGeneratorMonitor();
```

Comment:
Will be reverted when `5718c83b7ba5622797ef6d6731f1106534b65195` is removed from the branch